app/assets/javascripts/views/data_portal/slides/ShareView.js

```diff
@@ -0,0 +1,67 @@
+(function (App) {
+  App.View.ShareView = App.Component.Slide.extend({
+
+    template: JST['templates/data_portal/slides/shareWidget'],
+
+    className: 'slide',
+
+    events: {
+      'click .js-back': '_onBack',
+      'click .js-copy': '_onCopy',
+      'blur  .js-copy': '_onBlur'
+    },
+
+    initialize: function (settings) {
```

Comment:
Useless. If you don't do anything here, just remove the declaration of `initialize`.